# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/arcticfox/hotfix-no-shlibsign.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- Arctic-Fox-45.3/toolkit/mozapps/installer/packager.py.vanilla	2025-04-16 15:35:47.217625151 +0200
+++ Arctic-Fox-45.3/toolkit/mozapps/installer/packager.py	2025-04-16 15:37:42.797894281 +0200
@@ -364,7 +364,7 @@
 
     # shlibsign libraries
     if launcher.can_launch():
-        if not mozinfo.isMac and buildconfig.substs.get('COMPILE_ENVIRONMENT'):
+        if False and buildconfig.substs.get('COMPILE_ENVIRONMENT'):
             for lib in SIGN_LIBS:
                 libbase = mozpath.join(respath, '%s%s') \
                     % (buildconfig.substs['DLL_PREFIX'], lib)
